src/backend/TrafficCourtsApi/ticket-worker/Program.cs

```diff
@@ -42,7 +44,13 @@ internal static void ConfigureServiceBus(IServiceCollection services, IConfigura
             var port = configuration.GetSection("SMTPServer")["Port"];
             var from = configuration.GetSection("Mail")["From"];
             var fromEmail = configuration.GetSection("Mail")["FromEmail"];
+            string allowedEmails = configuration.GetSection("ALLOWED_RECIPIENT_EMAILS").Value;
```

Comment:
On all these configuration items, we need to be more defensive, what if something is not configured? Or configured wrong? Should the app just crash? Should it log out that there is an error with the email configuration and emails will be disabled? Should it disable the handler?